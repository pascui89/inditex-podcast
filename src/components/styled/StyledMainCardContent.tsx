import {
    CardContent,
    styled,
  } from '@mui/material';

export const StyledMainCardContent = styled(CardContent)`
    max-width: 100%;
    display: flex;
    justify-content: center;
    align-items: center;
    flex-direction: column;
`;