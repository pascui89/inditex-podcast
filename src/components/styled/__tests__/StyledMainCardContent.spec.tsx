import React from 'react';
import { render } from '@testing-library/react';
import { StyledMainCardContent } from '../StyledMainCardContent';
import '@testing-library/jest-dom/extend-expect';

global.React = React;

describe('StyledMainCardContent', () => {

  it('should render', () => {
    const { container } = render(<StyledMainCardContent/>);
    expect(container).toMatchSnapshot();
  });
});
