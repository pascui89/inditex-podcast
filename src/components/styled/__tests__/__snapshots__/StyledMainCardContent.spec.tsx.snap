// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledMainCardContent should render 1`] = `
<div>
  <div
    class="MuiCardContent-root css-yh82xd-MuiCardContent-root"
  />
</div>
`;
